src/test/java/org/jabref/logic/search/DatabaseSearcherWithBibFilesTest.java

```diff
@@ -81,8 +78,7 @@ private BibDatabaseContext initializeDatabaseFromPath(Path testFile) throws Exce
         ParserResult result = new BibtexImporter(mock(ImportFormatPreferences.class, Answers.RETURNS_DEEP_STUBS), new DummyFileUpdateMonitor()).importDatabase(testFile);
         BibDatabaseContext databaseContext = result.getDatabaseContext();
 
-        when(FILE_PREFERENCES.shouldFulltextIndexLinkedFiles()).thenReturn(true);
-        when(FILE_PREFERENCES.fulltextIndexLinkedFilesProperty()).thenReturn(mock(BooleanProperty.class));
+        when(FILE_PREFERENCES.fulltextIndexLinkedFilesProperty()).thenReturn(new SimpleBooleanProperty(true));
```

Comment:
Mocked preferences are not a constant, but they are modified just here with the when statement. Remove the final in bar declaration and use snake case for the car name, so the variable name is not misleading.

Reply:
The other uppercase constants need to be checked here too. @koppor, wdyt?

Reply:
Updated the preferences variables in [`84a2754` (#11542)](https://github.com/JabRef/jabref/pull/11542/commits/84a275492d31a008980b3f95bd617e3d3b0c6803).